dev-docs/bidders/lucead.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: bidder
+title: Lucead adapter
+description: Prebid Lucead Bidder Adapter
+biddercode: lucead
+tcfeu_supported: false
+gvl_id: none
+usp_supported: false
+coppa_supported: false
+schain_supported: false
+dchain_supported: false
+media_types: banner
+safeframes_ok: true
+deals_supported: true
+floors_supported: true
+fpd_supported: true
+pbjs: true/false
+pbs: true/false
+prebid_member: true/false
+ortb_blocking_supported: false
+privacy_sandbox: paapi
+sidebarType: 1
+---
+### Note
+
+The Lucead Bidding adapter requires setup before beginning. Please contact us at [aymeric@lucead.com](mailto:aymeric@lucead.com).
```

Comment:
You're not allowed to refer to a specific individual at a company. People leave. Please supply a group alias, e.g. "sales@lucead.com"